src/controls.py

```diff
@@ -82,17 +82,17 @@ class Controls(Control, Enum):
     Enum which groups all Control instances of the game.
     """
 
-    UP = (pygame.K_UP, "Move Up")
-    DOWN = (pygame.K_DOWN, "Move Down")
-    LEFT = (pygame.K_LEFT, "Move Left")
-    RIGHT = (pygame.K_RIGHT, "Move Right")
-
-    USE = (pygame.K_SPACE, "Use Tool")
-    NEXT_TOOL = (pygame.K_TAB, "Cycle Tools")
-    NEXT_SEED = (pygame.K_LSHIFT, "Cycle Seeds")
-    PLANT = (pygame.K_RETURN, "Plant Current Seed")
-    INTERACT = (pygame.K_i, "Interact")
-    EMOTE_WHEEL = (pygame.K_e, "Toggle Emote Wheel")
+    UP = (pygame.K_w, "Move Up")
+    DOWN = (pygame.K_s, "Move Down")
+    LEFT = (pygame.K_a, "Move Left")
+    RIGHT = (pygame.K_d, "Move Right")
+
+    USE = (1, "Use Tool")                   # Left Mouse Button
+    NEXT_TOOL = (pygame.K_q, "Cycle Tools")
+    NEXT_SEED = (pygame.K_e, "Cycle Seeds")
+    PLANT = (3, "Plant Current Seed")       # Right Mouse Button
+    INTERACT = (pygame.K_r, "Interact")
+    EMOTE_WHEEL = (pygame.K_c, "Toggle Emote Wheel")
```

Comment:
That change is kind of unfortunate. I mean, the default keybinds to arrow keys were universal since the arrow keys are always at the same position, which is not the case for the WASD keys. Also, why did you set the plant key to just the value 3?

Reply:
True, i'm not using WASD but ZQSD myself, it's just because i find easier to change only right left down keys for each new fetch ( also using only keyboard don't give me good user experience ). Should I reset all as before ? Did you try with mouse click actions and left hand movement ?

I left comments, 1 means left mouse button 3 means right mouse button (maybe a pygame events exist)

Reply:
In the case of movement defaults, these should be reverted. As for mouse buttons, you should probably put these values in constants so other collaborators understand their meaning better